tests/dynamics/test_bin_recapture.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+"""Test for bin_recapture function
+    in physics/dynamics.py
+"""
+######## Imports ########
+#### Standard Library ####
+import time
+#### Third-Party ####
+import numpy as np
+#### Homemade ####
+#### Local ####
+from mcfacts.physics.dynamics import bin_recapture
+
+######## Setup ########
+INPUT_DATA = {
```

Comment:
Please utilize the `INPUT_PARAMETERS` dictionary in `McFACTS/conftest.py` for these values.